core/trino-main/src/test/java/io/trino/sql/query/TestRowFilter.java

```diff
@@ -626,7 +626,7 @@ public void testInsert()
     }
 
     /**
-     * Like {@link #testInsert()} ()} but using the MERGE statement.
+     * Like {@link #testInsert()} but using the MERGE statement.
```

Comment:
Unrelated change?

Reply:
It's a separated commit. Let me extract it into another PR.